docs/installation.md

```diff
@@ -98,14 +98,15 @@ configuration YAML file.
 
 Find more information in the [routing rules documentation](routing-rules.md).
 
-
 ### Configure logging <a name="logging">
 
-Path to `log.properties` must be set via `log.levels-file` JVM options
-like `-Dlog.levels-file=etc/log.properties`.
+To configure the logging level for various classes, specify the path to the 
+`log.properties` file using the `log.levels-file` JVM options, such as 
+`-Dlog.levels-file=etc/log.properties`, when the logging type is `external`.
 
-Use the `log.*` properties from the [Trino logging properties
-documentation](https://trino.io/docs/current/admin/properties-logging.html) for further configuration.
+For additional configurations, use the `log.*` properties from the [Trino 
+logging properties documentation](https://trino.io/docs/current/admin/properties-logging.html) and specify the path to `log.config` 
+file using the `config` JVM options, such as `-Dconfig=var/log.config`.
```

Comment:
I would rather follow the Trino approach of having on config.properties file and just all that to use log properties .. and any others.

Reply:
oh so instead of two files (log.properties, log.config), you mean a single `config.properties` file?

Reply:
no... use same as in trino .. but this might be part of the airlift refacot

have log.properities for the log level for different packages
and config.properties for all config, including logging config